build.gradle

```diff
@@ -177,23 +177,10 @@ task tarGraal(type: Exec) {
 
     doFirst {
         if (environment.JVMCI_HOME.contains('openjdk1.8')) {
-            commandLine 'tar', 'czf', "$graalVMTar", '-C', getGraalBuildFolder.graalVMDir,
-                    "--exclude=./bin",
-                    "--exclude=./lib",
-                    "--exclude=./man",
-                    "--exclude=./sample",
-                    "--exclude=*src.zip",
-                    '.'
+            commandLine 'tar', 'czf', "$graalVMTar", '-C', getGraalBuildFolder.graalVMDir, '.'
```

Comment:
the two branches are the same

Reply:
also, if that means `JVMCI_HOME` env is no longer used then remove it

Reply:
did you forget to push the changes?

Reply:
Yes.

Reply:
I have pushed the commit that was missing.